evcxr/src/eval_context.rs

```diff
@@ -225,7 +225,7 @@ impl EvalContext {
                                 .external_deps
                                 .entry(crate_name.clone())
                                 .or_insert_with(|| {
-                                    ExternalCrate::new(crate_name.clone(), "\"*\"".to_owned())
+                                    ExternalCrate::new(crate_name.replace("_", "-").clone(), "\"*\"".to_owned())
```

Comment:
What if both the rustc crate name and crates.io package name are using `_`, like [`rand_core`](https://crates.io/crates/rand_core)?

Reply:
Ahh, I wrongly interpreted `_` and `-` as exchangeable which isn't the case. Will push a fix for this soon, thanks!

Reply:
So, interestingly even the crates.io-API doesn't make a difference between hyphens and underscores, but the `Cargo.toml` does. I updated the patch to check for the actual crate's name if an underscore exists. Thanks for your feedback!